tools/ci_build/build.py

```diff
@@ -400,6 +400,9 @@ def parse_arguments():
         "--use_openvino", nargs="?", const="CPU_FP32",
         type=_openvino_verify_device_type,
         help="Build with OpenVINO for specific hardware.")
+    parser.add_argument(
```

Comment:
would really prefer you don't add another build option. we shouldn't have more than 1 build option per EP.
it would be better if you can add support for another argument to --use_openvino instead. 
something like
--use_openvino CPU_FP32 NO_PARTITION